projects/packages/stats-admin/src/class-admin-post-list-column.php

```diff
@@ -112,7 +112,7 @@ public function add_stats_post_table_cell( $column, $post_id ) {
 
 				?>
 				<a href="<?php echo esc_url( $stats_post_url ); ?>"
-					title="<?php echo esc_html__( 'View stats for this post', 'jetpack-stats-admin' ); ?>">
+					title="<?php echo esc_html__( 'Views stats for last 30 days', 'jetpack-stats-admin' ); ?>">
```

Comment:
This implements what @ederrengifo requested on paYJgx-5Nz-p2#comment-5819.

I'm a bit concerned it's confusing - is the tooltip describing the information on screen (last 30 days), or on the link destination. I can't think of a nice way of fixing that, except to say something like "Views for the last thirty days. Click for detailed stats".

At the least though, I'd change it to "Views for the last thirty days", I think it reads better. "Views stats" sounds a bit off.

Approving, because I don't need to review wording changes.